src/pages/SignupPage.tsx

```diff
@@ -1,19 +1,24 @@
 import React, { useState } from "react";
-import { Redirect } from "react-router";
+import { Link } from "react-router-dom";
 
 import useFormInput from "../hooks/useFormInput";
+import AuthenticationForm from "../components/AuthenticationForm";
+import LayoutContainer from "../components/LayoutContainer";
+import { Paths } from "../constants";
+import { UnauthenticatedPageProps, User } from "../types";
 
-export default function SignupPage(): JSX.Element {
-  const email = useFormInput("");
-  const password = useFormInput("");
-  const [signedUp, setSignedUp] = useState(false);
+const SignupPage = ({
+  setSessionUser
+}: UnauthenticatedPageProps): JSX.Element => {
+  const { value: emailValue, onChange: onEmailChange } = useFormInput("");
+  const { value: passwordValue, onChange: onPasswordChange } = useFormInput("");
 
   function handleOnSubmit(e: React.FormEvent<HTMLFormElement>) {
     e.preventDefault();
 
     fetch("/signup", {
       method: "POST",
-      body: JSON.stringify({ email: email.value, password: password.value }),
+      body: JSON.stringify({ email: emailValue, password: passwordValue }),
       headers: {
```

Comment:
probs worth extracting some network utilities if we haven't done this already so you could do like `network.post(url, body)`

Reply:
I think this also makes sense in a separate PR? Agreed that fetch doesn't handle post very well out of the box